usermods/LDR_Dusk_Dawn_v2/usermod_LDR_Dusk_Dawn_v2.h

```diff
@@ -0,0 +1,126 @@
+#pragma once
+#include "wled.h"
+
+class LDR_Dusk_Dawn_v2 : public Usermod {
+  private:
+    // Defaults
+    bool ldrEnabled = false;
+    int ldrPin = 34; //A2 on Adafruit Huzzah32
+    int ldrThresholdMinutes = 5; // How many minutes of readings above/below threshold until it switches LED state
+    int ldrThreshold = 1000; // Readings higher than this number will turn off LED. 
+    int ldrOnPreset = 1; // Default "On" Preset
+    int ldrOffPreset = 2; // Default "Off" Preset
+
+    // Variables
+    bool ldrEnabledPreviously = false; // Was LDR enabled for the previous check? First check is always no.
+    int ldrOffCount; // Number of readings above the threshold
+    int ldrOnCount; // Number of readings below the threshold
+    int ldrReading; // Last LDR reading
+    int ldrLEDState; // Current LED on/off state
+    unsigned long lastMillis = 0;
+    static const char _name[];
+
+  public:
+    void setup() {
+    }
+
+    void loop() {
+      // Only update every 10 seconds
+      if (millis() - lastMillis > 10000) {      
+        if (ldrEnabled == true) {
```

Comment:
loop should do nothing if not enabled: `if (!ldrEnabled) return;`

Reply:
usermod, so ... 😄